app/scss/_call-to-action.scss

```
@@ -0,0 +1,32 @@
+.call-to-action {
+    background-color: #5b605f;
+    color: white;
+    & .call-to-action__wrapper {
```

Ampersand (`&`) is redundant here. In short, these both are working similarly:
```
.a {
  .b {
    color: white;
  }
}
```

```
.a {
  & .b {
    color: white;
  }
}
```
and will produce `.a .b { color: white; }`. Maybe you wanted to do it in this way:
```suggestion
&__wrapper {
```
?

---

ok. i changed that